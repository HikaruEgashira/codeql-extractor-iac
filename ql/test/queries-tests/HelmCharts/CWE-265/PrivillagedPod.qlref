security/HelmCharts/CWE-265/PrivillagedPod.ql