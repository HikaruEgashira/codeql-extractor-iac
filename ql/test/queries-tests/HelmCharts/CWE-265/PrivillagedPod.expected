| Chart.yml:6:15:6:18 | true | privileged enabled |
| Chart.yml:32:21:32:24 | true | privileged enabled |
